core/test/servico/cst04Test.java

```diff
@@ -60,7 +60,7 @@ public void tearDown() {
     public void testeEfetuarAlocação_Confirmar() throws ParseException
     {
         LoginService loginService = new LoginService();
-        assertTrue(loginService.VerificarUsuarioESenha("Administrador", "123456"));
+        assertTrue(loginService.verificarUsuarioESenha("Administrador", "123456"));

```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [Line is longer than 80 characters (found 83).](https://www.codacy.com/app/calebepb/substituicao/file/2734604411/issues/source?bid=3308597&fileBranchId=3351368#l63)